contracts/Company/Extras/ExitController.sol

```diff
@@ -0,0 +1,281 @@
+pragma solidity 0.4.26;
+
+import "../../Math.sol";
+import "../../Universe.sol";
+import "../../Agreement.sol";
+import "../../Reclaimable.sol";
+
+import "../IEquityTokenController.sol";
+import "../IEquityToken.sol";
+import "../../ETO/IETOCommitment.sol";
+import "../../Standards/IContractId.sol";
+
+/*
+To test:
+* Disbursal by nominee
+*
+ */
+
+contract ExitController is
+    KnownInterfaces,
+    Reclaimable,
```

Comment:
Let's disallow reclaiming `EQUITY_TOKEN` completely and `EURO_TOKEN` up until we reach either `ManualPayoutResolution` or a new `Closed` state